src/components/CookieWarning.js

```diff
@@ -34,8 +34,9 @@ const CookieWarning = () => {
     signOut()
   }
   return !cookiesAccepted && div({
+    role: 'banner',
     style: {
-      position: 'fixed', height: 100, bottom: 0, width: '100%',
+      flex: 0, height: 100, width: '100%',
```

Comment:
I ran this by UX, this is fine, but our UX team would want some other changes as well. Namely, the thought is:
Instead of `borderTop` on line 41, instead that should be `borderBottom`
For the background color, can we make it `colors.dark(0.15)`, just so it pops a bit more as being not part of the main site body?

Reply:
Sure! Both great suggestions